src/api/common/connection.js

```diff
@@ -96,24 +67,26 @@ class Connection extends EventEmitter {
     return this._ws !== null;
   }
 
+  isConnected() {
```

Comment:
I kind of like the consistency of all public methods being functions; so I'm going to make `state` private and keep this as a function. The function notation makes it more obvious that it's not a settable property.